src/main/java/com/novoda/notils/string/QueryUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.notils.string;
+
+import java.util.Arrays;
+
+public class QueryUtils {
+
+    /**
+     * Creates a string to be used as a placeholder in {@link android.content.ContentResolver} operations selection.
+     * This will allow to use more selection arguments and have them replaced when using the IN operator.
+     *
+     * @param size The number of selection arguments
+     * @return Example: for size == 3 : "?, ?, ?"
+     */
+    public static String createSelectionPlaceholdersOfSize(int size) {
+        String[] questionMarks = new String[size];
```

Comment:
what about something like:
```
int sizeOfResult = size * 3 - 2;
char[] result = new char[sizeOfResult];
for (int i = 0; i < size; i++) {
  result[i] = '?';
  result[i +1] = ',';
  result[i + 2] = ' ';
}
if (i > 0) {
  result[sizeOfResult - 1] = '?';
}
return new String(result);
```

it creates only two objects, instead of size + 5, and is still understandable and simple.